add tutorial chapter for spatial data [WIP]

pat-s opened the pull request:
see https://github.com/mlr-org/mlr/pull/1891 (waiting for approval)

Commit:
update auto-generated tutorial pages [ci skip]

Commit:
update auto-generated tutorial pages [ci skip]

Commit:
update auto-generated tutorial pages [ci skip]

Commit:
update auto-generated tutorial pages [ci skip]

Commit:
update auto-generated tutorial pages [ci skip]

pat-s committed:
rebase

Merge branch 'gh-pages' of https://github.com/pat-s/mlr-tutorial into gh-pages

# Conflicts:
#	devel/html/advanced_tune/index.html
#	devel/html/benchmark_experiments/index.html
#	devel/html/cost_sensitive_classif/index.html
#	devel/html/create_filter/index.html
#	devel/html/create_measure/index.html
#	devel/html/feature_selection/index.html
#	devel/html/hyperpar_tuning_effects/index.html
#	devel/html/index.html
#	devel/html/mkdocs/search_index.json
#	devel/html/multilabel/index.html
#	devel/html/nested_resampling/index.html
#	devel/html/parallelization/index.html
#	devel/html/preproc/index.html
#	devel/html/resample/index.html
#	devel/html/roc_analysis/index.html
#	devel/html/sitemap.xml
#	devel/html/tune/index.html
#	devel/mlr_tutorial.zip

pat-s committed:
merge upstream

Merge branch 'gh-pages' of github.com:mlr-org/mlr-tutorial into gh-pages

# Conflicts:
#	devel/html/advanced_tune/index.html
#	devel/html/benchmark_experiments/index.html
#	devel/html/cost_sensitive_classif/index.html
#	devel/html/create_filter/index.html
#	devel/html/create_measure/index.html
#	devel/html/feature_selection/index.html
#	devel/html/hyperpar_tuning_effects/index.html
#	devel/html/index.html
#	devel/html/mkdocs/search_index.json
#	devel/html/multilabel/index.html
#	devel/html/nested_resampling/index.html
#	devel/html/parallelization/index.html
#	devel/html/preproc/index.html
#	devel/html/resample/index.html
#	devel/html/roc_analysis/index.html
#	devel/html/sitemap.xml
#	devel/html/tune/index.html
#	devel/mlr_tutorial.zip

pat-s committed:
re-add spatial.Rmd

pat-s committed:
update spatial.Rmd

pat-s committed:
add spatial to mkdocs

pat-s committed:
add images

pat-s committed:
pull upstream

Merge branch 'gh-pages' of github.com:mlr-org/mlr-tutorial into gh-pages

# Conflicts:
#	devel/html/advanced_tune/index.html
#	devel/html/benchmark_experiments/index.html
#	devel/html/cost_sensitive_classif/index.html
#	devel/html/create_filter/index.html
#	devel/html/create_measure/index.html
#	devel/html/feature_selection/index.html
#	devel/html/hyperpar_tuning_effects/index.html
#	devel/html/index.html
#	devel/html/multilabel/index.html
#	devel/html/nested_resampling/index.html
#	devel/html/performance/index.html
#	devel/html/preproc/index.html
#	devel/html/resample/index.html
#	devel/html/roc_analysis/index.html
#	devel/html/search/search_index.json
#	devel/html/sitemap.xml
#	devel/html/tune/index.html
#	devel/mlr_tutorial.zip

pat-s committed:
harmonize with gh-pages

Merge branch 'gh-pages' into spatial

# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.

jakob-r commented on .gitignore:
```diff
@@ -3,3 +3,5 @@ cache
 html/.DS_Store
 .Rproj.user
+full_code_src
+devel/
```
you must not ignore devel. I understand that you don't want to commit things in here. It will be pushed by travis. If you ignore it the autobuild won't work. If I force push devel there it will end up in the repo regardless of the gitignore...

pat-s replied:
OK :+1:

jakob-r commented on mkdocs.yml:
```diff
@@ -41,6 +41,7 @@ pages:
     - 'Classifier Calibration Plots': 'classifier_calibration.md'
     - 'Hyperparameter Tuning Effects': 'hyperpar_tuning_effects.md'
     - 'Out-of-Bag Predictions': 'out_of_bag_predictions.md'
+    - 'Spatial': 'spatial.md'
```
If your headline ist "Handling of Spatial Data" it should have the same name here. Also please name the md file `handling_of_spatial_data.md`.

pat-s replied:
ok :+1:

jakob-r reviewed (changes requested):
I guess the text is WIP, so I did not read.

pat-s commented:
> I guess the text is WIP, so I did not read.

Yes but it is close to be finished. However, it won't be merged until https://github.com/mlr-org/mlr/pull/1891 has been merged.

pat-s committed:
remote devel/ from .gitignore

pat-s committed:
rename to 'handling_of_spatial_data'

pat-s committed:
update gitignore, delete devel

pat-s committed:
fix typo

pat-s committed:
re-add devel from gh-pages

pat-s committed:
fix devel/

pat-s committed:
delete mkdocs stuff

Merge branch 'gh-pages' into spatial

# Conflicts:
#	devel/html/search/search_index.json
#	devel/mlr-tutorial.pdf
#	devel/mlr_tutorial.zip

pat-s committed:
delete search_index.json

pat-s committed:
delete pdf version and .zip

pat-s committed:
delete gitignore

pat-s committed:
add cheatsheets

pat-s committed:
update .Rmd

Commit:
fix typo

pat-s commented:
see #131 for a clean PR
